front/src/applications/stdcm/components/RunningTime.tsx

```diff
@@ -23,8 +24,10 @@ export function withProps<T>(Component: ComponentType<T>) {
 
 function RunningTime({ dispatch = noop }: RunningTimeProps) {
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
+  const { getMaximumRunTime } = useOsrdSelectors() as stdcmConfSelectorsType;
   const maximumRunTime = useSelector(getMaximumRunTime);
   const [isRtBelowCap, setIsRtBelowCap] = useState(true);
+  const { updateMaximumRunTime } = useOsrdActions() as stdcmConfSliceActionsType;
```

Comment:
Like most of my comments here, just a personnal preference for readability :)
```suggestion
  const { getMaximumRunTime } = useOsrdSelectors() as stdcmConfSelectorsType;
  const { updateMaximumRunTime } = useOsrdActions() as stdcmConfSliceActionsType;
  const maximumRunTime = useSelector(getMaximumRunTime);
  const [isRtBelowCap, setIsRtBelowCap] = useState(true);
```

Reply:
Done